djangocms_navigation/forms.py

```diff
@@ -20,10 +21,11 @@ class Meta:
 
 class MenuContentForm(forms.ModelForm):
     title = forms.CharField(label=_("Menu Title"), max_length=100)
+    language = forms.ChoiceField(choices=tuple(get_language_tuple()))
```

Comment:
Last change, we're going to need to filter the list of languages to only those that are available for the site: https://github.com/divio/django-cms/blob/develop/cms/utils/i18n.py#L94

Reply:
What could happen is that a language could be added that could never be used on a site.